docs.html

```diff
@@ -270,26 +270,126 @@ <h6>H6-Heading</h6>
-              style="width: 490px; height: 313px; border:0; transform: scale(1); overflow:hidden;"
-              sandbox="allow-scripts allow-same-origin">
-            </iframe>
-          </div>
+      </div>
+
+
+      <div class="code-frame">
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-dark&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=7px&ph=8px&ln=false&fl=1&fm=dm&fs=13px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Ch1%253EH1-Heading%253C%252Fh1%253E%250A%2520%253Ch2%253EH2-Heading%253C%252Fh2%253E%250A%2520%253Ch3%253EH3-Heading%253C%252Fh3%253E%250A%2520%253Ch4%253EH4-Heading%253C%252Fh4%253E%250A%2520%253Ch5%253EH5-Heading%253C%252Fh5%253E%250A%2520%253Ch6%253EH6-Heading%253C%252Fh6%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%253Cdiv%2520class%253D%2522h2%2522%253EH2-Heading%253C%252Fdiv%253E%250A%2520%2520%250A%2520%2520%253C--Other%2520Classes---%253E%250A%2520%2520%253Cp%2520class%253D%2522text-strike%2522%253EStrike%2520through%2520line.%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522text-primary%2522%253EText%2520with%2520primary%2520%28dark%29%2520color.%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522text-danger%2522%253EText%2520with%2520danger%2520%28dark%29%2520color.%253C%252Fp%253E"
+          style="width: 100%; height: 313px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+      </div>
     </section>
 
     <!-- Components Description-->
+
+    <!-- Avatars -->
+
     <section id="Avatars">
       <h4 class="section-heading">Avatars</h4>
-      <p></p>
+      <p>Avatar are images which can be used to show user's profile image. Use base class <span class="code-highlight">
+          .avatar</span> to generate a avatar.</p>
+
+      <h5>Image Avatars</h5>
+      <p>Image avatars can be created with class <span class="code-highlight"> .avatar-img</span></p>
+
+      <div class="component-display">
+        <figure class="avatar avatar-md">
+          <img class="avatar-img" src="./assets/images/rengoku.jpg" alt="">
```

Comment:
Add an alt attribute to the images used in documentation.In this case you can mention anime-character or Demo Image😅.

Reply:
fixed - d024da009973ecafa0fe044faac36624c2778267